### PR TITLE
Resolve "babel-plugin-" name before trying the plain name.

### PR DESCRIPTION
**Pull request by jquense**
Otherwise you get situations like the following: `plugins: [ 'object-assign']` resolves to the module `object-assign` (an Object.assign polyfill) instead of the intended plugin packaged named: `babel-plugin-object-assign`.

### CODE REVIEW

**Comment by sebmck** [comment]
Thanks! Spelling mistake in the commit message though. `plane` -> `plain`.

**Comment by jquense** [comment]
fixed!

File: vendor/traceur [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit e484618da417b5fd6889395bd779d5b3e08c43cb
+Subproject commit c43912d121cb42b84b870565338f7e35d6a61af2
```
Comment (sebmck) [inline vendor/traceur]: Shouldn't be included in the commit as this may break the tests.

**Comment by jquense** [comment]
blah sorry again, should be clean now

**Comment by sebmck** [comment]
Thanks!

**Comment by sebmck** [comment]
Sorry for the delay, released as of 5.4.4. Thanks!
